src/Relude/DeepSeq.hs

```diff
@@ -26,27 +26,58 @@ import Relude.Monad (MonadIO, liftIO, (<$!>))
 
 import qualified Control.Exception.Base (evaluate)
 
+{- | Lifted alias for 'Control.Exception.Base.evaluate' with clearer name.
 
--- | Lifted alias for 'Control.Exception.Base.evaluate' with clearer name.
+>>> :sprint list
```

Comment:
Variable `list` should be defined before this `:sprint` as I wrote in the comment here:

* https://github.com/kowainik/relude/pull/186/files#r323600472

Reply:
oh sorry my bad was looking for a specific `list` function on Stackage 😂